import "MDropList.proto";
message ParamUnit
{
	optional int32 type = 1;	//参数类型
	optional string param = 2;	//参数
}

message MailData
{
	optional string id 			= 1;//唯一id
	optional string createtime 	= 2;//创建时间
	optional string	title		= 3;//标题
	optional string content		= 4;//内容
	optional string name		= 5;//发件人或者收件人
	optional string uid			= 6;//发件人或者收件人uid
	repeated ParamUnit paramlist = 7;//参数列表
	optional MDropList 	reward	= 8;//奖励列表
	optional bool		isget	= 9;//是否领取
	optional bool		isread	= 10;//是否读取
}

message DseMailOpt
{
	optional int32	res			= 1;//
	optional int32	type 		= 2;//1请求邮件 2领取 3发邮件 4邮件已读
	optional int32  mailtype 	= 3;//邮件类型(1奖励邮件 2通知邮件 3战报 4收件箱 5发件箱)
	repeated MailData 	maillist 	= 4;//某邮件类型的列表
	repeated int32		newcounts	= 5;//未读邮件数量 奖励 系统 战斗 收件箱 发件箱
	optional MDropList	drops		= 6;//领取的奖励
}



